tests/test_examples.py

```diff
@@ -7,11 +7,14 @@
 import glob
 import os
 import shutil
+import sys
 import tempfile
 import unittest
 from pathlib import Path
 from typing import ClassVar, List
 
+from tests import utils
+
 
```

Comment:
I linted with `isort` this file and that was what suggested.
I thought the list should be like that:
```
import glob
import os
import shutil
import sys
import tempfile
import unittest

from pathlib import Path
from typing import ClassVar, List
from tests import utils
```
but isort fixes that to the current state.